<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ketan" failures="1" tests="2" name="error" time="8.147" errors="0" timestamp="2024-02-26T15:15:07 IST">
  <testcase classname="Selenium.Tests.error" name="WrongCredentials" time="0.989">
    <skipped/>
  </testcase> <!-- WrongCredentials -->
  <testcase classname="Selenium.Tests.error" name="WrongCredentials" time="0.477">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Selenium.Tests.error.WrongCredentials(error.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- WrongCredentials -->
</testsuite> <!-- error -->
